src/Cpanel/PluginActions.php

```diff
@@ -74,9 +75,9 @@ public function getUserConfig()
             $userConfigContent = @file_get_contents('./config.json');
 
             //Need to set an empty array for merge into config
-            if ($userConfigContent === false) {
-                $this->userConfig = [];
-            } else {
+            $this->userConfig = [];
+            //If we did find a config decode it
+            if ($userConfigContent) {
```

Comment:
Don't comment what is obvious like 
```
//The width of the line is 2
lineWidth = 2;
```

Write comments explaining "why" rather than "what" and only write it when it's needed. The "what" should be understandable by reading the code itself